src/app/core/services/auth.guard.ts

```diff
@@ -0,0 +1,8 @@
+import { inject } from '@angular/core';
+import { AuthState } from "@src/app/state/auth.state";
+import { Store } from "@ngxs/store";
+
+export const authGuard = (): boolean => {
+  const store = inject(Store);
+  return store.selectSnapshot(AuthState.isAuthenticated);
```

Comment:
This only allows or denies access to a page. What I would like to have is: if user is logged in, allow page access, otherwise – redirect them to Log in page.
Redirect should not be done with `Router.navigate`, do it with `Router.createUrlTree`.